### ISSUE TITLE
Configuration for already created websites

### ISSUE THREAD

**Issue by s-belichenko-sold** [vccw-team/vccw#255]
I'm looking for a way to quickly and correctly use your project for already created websites. In other words,  I often need to install everything in the box, except Wordpress itself. Tell me, please, is there any simple way to cancel the installation of Wordpress, plug-ins to it and other wordpress things?

**Comment by iamandrewluca** [comment]
@s-belichenko-sold I just did this. Go to 
```
provision/playbook.yml
```
and comment third line 
```
---
- include: playbooks/commands.yml
- include: playbooks/middleware.yml
# - include: playbooks/wordpress.yml
```
Do a `vagrant up`. Now your VM is up and running, now put your site in `/var/www/html` and import your database according to your `wp-config.php`

**Comment by s-belichenko-sold** [comment]
@iamandrewluca yep, I'm already tried this way. Yes, this is the obvious way, I've already edited the configuration for my task manually. But I'm talking about the fact that it would be nice to be able to turn off and turn on absolutely all things. More fine-tuning. So far, this possibility has been done only for some things.

Just disabling some configuration file is not a very fine tuning :)

**Comment by iamandrewluca** [comment]
For me it worked like a charm :)

**Comment by mbwhite** [comment]
I'm also using VCCW in a similar manner. My goal is to take a backup of the site, (Softcalculus is the control panel my host gives me) and run this locally. To confirm the backup is good, test changes, and also be an offline copy to help train uses on the site. 

Here's the steps aproximately
- `vagrant up` and then `vagrant ssh`
- Then I drop the created database, and recreate it based on my backup copy
- I adjust the wp config to match the user/name password.
- I then add
```
define('WP_HOME','http://192.168.33.10');
define('WP_SITEURL','http://192.168.33.10');
``` 
to the `wp-confg.php` to make the site behave correctly as it's on a different host.

It's not as elgant as I would like; but it works at the moment.  I would like to try and make this slicker and contribute back.
